tracer/src/Datadog.Trace/Activity/Handlers/DefaultActivityHandler.cs

```diff
@@ -21,7 +22,7 @@ public bool ShouldListenTo(string sourceName, string? version)
 
         public void ActivityStarted<T>(string sourceName, T activity)
             where T : IActivity
-            => ActivityHandlerCommon.ActivityStarted(sourceName, activity, tags: null, out _);
+            => ActivityHandlerCommon.ActivityStarted(sourceName, activity, tags: new OpenTelemetryTags(), out _);
```

Comment:
Can you double check that all callers of `ActivityHandlerCommon.ActivityStarted` are updated? I think that the Azure Service Bus handler may need to be updated to carry some of the new tags

Reply:
So what I opted to do was update `ActivityHandlerCommon.ActivityStarted` to take `OpenTelemetryTags` instead of an `ITags`.
Then I made the `AzureServiceBusTags` inherit `OpenTelemetryTags` and that seemed to get the mapping working correctly for the operation name.

I'm not super sure whether or not that'll cause issues, I did have it complain about a duplicate `span.kind` tag, so I removed the `[Tag]` attribute on the `AzureServiceBusTags.SpanKind` property and just override `OpenTelemetryTags.SpanKind`, which seemed to do the trick.